accounts/tests.py

```diff
@@ -1,3 +1,30 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from .models import UserAccount
 
-# Create your tests here.
+
+# test account to reduce redundancy
+@pytest.fixture
+def test_account(db):
+    username = 'testuser'
+    password = 'lvkmdgv32fdbf1b'
+    email = 'testuser@test.com'
+    new_user = User.objects.create_user(
+        username=username, password=password, email=email
+    )
+    return new_user
+
+
+# Create a user and check if a UserAccount instance was created
+@pytest.mark.django_db()
+def test_user_create(test_account):
+    assert User.objects.count() == 1
+    assert UserAccount.objects.count() == 1
+
+
+# Find user in database by user id, username or email
+@pytest.mark.django_db()
+def test_user_find(test_account):
+    assert User.objects.get(id=test_account.id)
```

Comment:
You can skip this test, the next two (username and email) should be enough

Reply:
Removed it and moved the comments